src/xcode/ENA/ENA/Source/Scenes/App/AppNavigationController.swift

```diff
@@ -39,10 +39,21 @@ class AppNavigationController: UINavigationController {
 
 	override func viewWillDisappear(_ animated: Bool) {
 		super.viewWillDisappear(animated)
-		
+
 		scrollViewObserver?.invalidate()
 	}
 
+	override func viewDidAppear(_ animated: Bool) {
+		super.viewDidAppear(animated)
+
+		if let topViewController = topViewController {
+			// Since we invalidate the observer on `viewWillDisappear()`, we need to establish it again here.
+			// Else the opacity is not set correctly upon returning to this screen.
+			// Ex. this happens when the FAQ page is presented.
```

Comment:
thank you so much for adding comments to your pull request. it is GREATLY appreciated. 👏